lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -337,10 +337,13 @@ a.edit-link:link, a.edit-link:visited, a.section-merge-link:link, a.section-merg
     -moz-border-radius: 7px;
     border-radius: 7px;
     border: 1px solid #222;
-    display:inline-block;
     vertical-align: top;
 }
 
+div.Mrphs-portalWrapper .column {
```

Comment:
A comment really helps explain why you're putting this property in it's own section.